assets/package.json

```diff
@@ -0,0 +1,140 @@
+{
+	"scripts": {
+		"build": "npm run build:production",
+		"prebuild:dev": "npm run remove-dist",
+		"build:dev": "webpack --mode=development --devtool cheap-module-eval-source-map --output-pathinfo",
+		"prebuild:production": "npm run remove-dist",
+		"build:production": "webpack --mode=production",
+		"build:test": "npm run build:production -- --include-tests",
+		"watch": "npm run build:dev -- --watch",
+		"remove-dist": "rimraf ../dist"
+	},
+	"browserslist": [
+		"extends @wordpress/browserslist-config"
+	],
+	"dependencies": {
+		"@lit-labs/react": "^1.2.0",
+		"@material-ui/core": "^4.11.4",
+		"@material/button": "^2.3.0",
+		"@material/checkbox": "^2.3.0",
+		"@material/dialog": "^2.3.0",
+		"@material/form-field": "^2.3.0",
+		"@material/layout-grid": "^0.41.0",
+		"@material/linear-progress": "^1.1.0",
+		"@material/list": "^2.3.0",
+		"@material/menu": "^2.3.0",
+		"@material/radio": "^2.3.0",
+		"@material/react-card": "^0.15.0",
+		"@material/react-chips": "^0.15.0",
+		"@material/react-dialog": "^0.15.0",
+		"@material/react-select": "^0.15.0",
+		"@material/react-tab-bar": "^0.15.0",
+		"@material/react-text-field": "^0.15.0",
+		"@material/ripple": "^2.3.0",
+		"@material/select": "^2.3.1",
+		"@material/switch": "^2.3.0",
+		"@material/textfield": "^2.3.1",
+		"@material/theme": "^1.1.0",
+		"@material/web": "^1.0.0-pre.9",
+		"@popperjs/core": "^2.9.2",
```

Comment:
And this one, do we need this?

Reply:
Hmm, I wasn't sure about this but this [comment](https://github.com/google/site-kit-wp/pull/5338/files#r895542461) suggests it's not actually needed for `assets`, so I've removed it.